From 531563dc2877e7cc71337facec58942d5ded42c8 Mon Sep 17 00:00:00 2001
From: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
Date: Fri, 25 Jun 2021 09:33:15 -0500
Subject: [PATCH 1/1] recipes-phosphor/dbus: Add enumeration "SCM_FPGA" for
 version property
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit

Adds enumeration "SCM_FPGA" for version property to support SCM_FPGA
firmware update.

Signed-off-by: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
---
 xyz/openbmc_project/Software/Version.interface.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/xyz/openbmc_project/Software/Version.interface.yaml b/xyz/openbmc_project/Software/Version.interface.yaml
index 345e5b5..aa16a8b 100644
--- a/xyz/openbmc_project/Software/Version.interface.yaml
+++ b/xyz/openbmc_project/Software/Version.interface.yaml
@@ -38,3 +38,6 @@ enumerations:
         - name: PSU
           description: >
             The version is a version for a PSU.
+        - name: SCM_FPGA
+          description: >
+            The version is a version for a SCM FPGA.
--
2.17.1

